lib/internal/Magento/Framework/App/ProductMetadata.php

```diff
@@ -25,7 +25,7 @@ class ProductMetadata implements ProductMetadataInterface, DistributionMetadataI
     /**
      * Magento product name
      */
-    public const PRODUCT_NAME  = 'Magento';
+    public const PRODUCT_NAME  = 'Mage-OS';
```

Comment:
This is intentionally left as `Magento` for compatibility reasons. Product Name = Magento; Distribution Name = Mage-OS. This way any code that checks if it's Magento(-compatible) gets a yes.

If any tests fail as a result of this being Magento rather than Mage-OS, we'll need to fix those instead.